### ISSUE TITLE
DC crashes when you try to open a comment dialog box for file-writing comments

### ISSUE THREAD

**Issue by joefowler** [usnistgov/dastard-commander#50]
We see this at the terminal:

```text
SEND SourceControl.ReadComment 0
Request: {'params': [0], 'method': 'SourceControl.ReadComment', 'id': 2}

Error: open /data/20181003/0000/comment.txt: no such file or directory
PANIC unhandled response.get(error)
Traceback (most recent call last):
  File "/home/pcuser/dastard-commander/writing.py", line 128, in comment
    dialog.setInputMode(QtWidgets.QInputDialog.InputMode.TextInput)
AttributeError: type object 'InputMode' has no attribute 'TextInput'
Aborted (core dumped)
```

Appears to be a simple bug in the dialog setup, but how did it ever work??

**Comment by ggggggggg** [comment]
It's probably my fault, though I don't remember enough to say why. Would have been when trying to fix #43

**Comment by joefowler** [comment]
Well, don't worry. I have a fix. Looks like you never tested the new design in the case that no comment.txt file previously exists.
